WarOfEmpires.Domain/Empires/BuildingDefinitionFactory.cs

```diff
@@ -248,6 +248,22 @@ public static class BuildingDefinitionFactory {
             );
         }
 
+        private static BuildingDefinition GenerateMarket() {
+            var descriptions = new ExpressionGenerator<string>("Your market allows you to send merchants off to sell your excess resources");
+            descriptions.Add(1, SequenceGeneratorFactory.GetGeneratorFunction((value) => $"Your market allows you to send merchants off to sell your excess resources; each merchant caravan can carry {value * 5000} resources"));
```

Comment:
Consider whether info on sales tax needs to be added here as well as on the market page

Reply:
Seems out of place here.